### ISSUE TITLE
Stuttering with Lenovo Explorer in Marvel Powers United Vr

### ISSUE THREAD

**Issue by SilentMosquito** [LibreVR/Revive#1047]
All other games through revive work well but I have a weird stutter when I play Marvel Powers United VR.

**Comment by FrameSkip** [comment]
Same here.  No matter what setting I use it performs poorly.  All other Revive apps work great.

**Comment by SilentMosquito** [comment]
@FrameSkip yep, and I have tried everything and can't get it working. Hoping for a fix in an update.

**Comment by MaxCancer** [comment]
i've an HP WMR VR pending hardware issue, can't start the game

**Comment by IMightBeBoman** [comment]
This also happens on my vive but doesn't happen on any other game run through revive very weird...

**Comment by dtdman420** [comment]
samsung odyssey here and im also getting constant stuttering in marvel powers united. Using revive and the game does not start at all with remixed

**Comment by SamCharette** [comment]
@MaxCancer restart your PC and/or reinstall marvel.  Doing both worked for me.  (HP set as well)

**Comment by BlackBoltagon** [comment]
The same here, have you server crashes too ?

**Comment by Dragoboss** [comment]
So for all of you....
If this happens only on one game, but not on any other Revive games... How big is the chance that Revive is the cause and thus that posting on the Revive issue tracker is going to resolve it?

I'll look into this. Since I also have a friend who owns this game and a Rift, and doesn't even get higher than 3 FPS...
The way I see it, I too have a little stutter but I think that's my computer lagging behind with this game and not Revive messing up.
I could be wrong. So I'll look into it.

**Comment by FrameSkip** [comment]
Sorry for the late reply but I don't think it's my PC.  I tried to set most of my settings very low and saw no noticeable improvement.  That said I guess some updates came that improved performance but I ended up refunding the game.
